ccomp_timer/test_apps/main/ccomp_timer_test.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "unity.h"
+#include "unity_test_runner.h"
+#include "esp_heap_caps.h"
+#include "esp_newlib.h"
+
+#define TEST_MEMORY_LEAK_THRESHOLD_DEFAULT -50
+static int leak_threshold = TEST_MEMORY_LEAK_THRESHOLD_DEFAULT;
+void set_leak_threshold(int threshold)
+{
+    leak_threshold = threshold;
+}
+
+static size_t before_free_8bit;
+static size_t before_free_32bit;
+
+static void check_leak(size_t before_free, size_t after_free, const char *type)
```

Comment:
Since this test is run in IDFv5.0 and above you can use utilities from `#include "unity_test_utils_memory.h"`
Eg.

``` c
unity_utils_setup_heap_record(80);
unity_utils_set_leak_level(128);
...
```

Reply:
Thanks, I have missed that. I'll update all the test apps to use unity_test_utils_memory.h.